src/pages/ApplicationsPage.tsx

```diff
@@ -2,14 +2,21 @@ import React from 'react';
 import ApplicationListView from '../components/ApplicationListView/ApplicationListView';
 import { HeadTitle } from '../components/HeadTitle';
 import NamespacedPage from '../components/NamespacedPage/NamespacedPage';
+import PageAccessCheck from '../components/PageAccess/PageAccessCheck';
 import { useQuickstartCloseOnUnmount } from '../hooks/useQuickstartCloseOnUnmount';
+import { ApplicationModel } from '../models';
+import { AccessReviewResources } from '../types';
 
 const ApplicationsPage = () => {
   useQuickstartCloseOnUnmount();
+  const accessReviewResources: AccessReviewResources = [{ model: ApplicationModel, verb: 'get' }];
```

Comment:
Not entirely sure about this, but for list pages, should we be using the `list` verb instead? Although we are actually doing a `watch` :thinking:

Reply:
Ah I see. For list page maybe we can add both `list`, `watch`